minecraft_servers/badhub/manifest.json

```diff
@@ -0,0 +1,46 @@
+{
+  "server_name": "badhub",
+  "nice_name": "BadHub",
+  "direct_ip": "mc.badhub.cz",
+  "server_wildcards": [
+    "%.badhub.cz"

```

Comment:
Doesn't look like a real wildcard
![image](https://github.com/user-attachments/assets/a88d3d53-5e5b-4a72-81c4-697cad1ac678)

Reply:
Still not a wildcard
https://i.imgur.com/PkUrfbL.png